package.json

```diff
@@ -71,6 +71,8 @@
     "gulp-uglify": "^2.0.0",
     "gulp-watch": "^4.3.5",
     "human-format": "^0.6.0",
+    "ip-range-generator": "^1.0.1",
```

Comment:
You need to reimplement the part you're using directly in xo-web as we do not currently support ES6 external deps.